test/ble_manager_test.dart

```diff
@@ -0,0 +1,16 @@
+
+import 'package:flutter_ble_lib/flutter_ble_lib.dart';
+import 'package:test/test.dart';
+
+void main() {
+  test('Ble manager must be singleton', () {
+    //given
+    final firstInstanceOfBlemanager = BleManager();
+
+    //when
+    final secondInstanceOfBleManager = BleManager();
+
+    //then
+    expect(secondInstanceOfBleManager, firstInstanceOfBlemanager);
```

Comment:
Different matcher should be used: https://api.flutter.dev/flutter/package-matcher_matcher/same.html